### PR TITLE
Feat: Image component

### PR DESCRIPTION
**Pull request by arulk8**
Desc: 
Added Image component

APP Link: https://confident-wozniak-3edef2.netlify.app/components/images/

### CODE REVIEW

**Review by sumitch29 — APPROVED** [review]
Looks Good.

File: components/Images/images.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  width: 6rem;
+  height: 6rem;
+  border-radius: 50%;
+}
+.img__round--md {
+  width: 10rem;
+  height: 10rem;
+  border-radius: 50%;
+}
+.img__round--lg {
```
Comment (harshitpaliwal95) [inline components/Images/images.css:33]: - img is shrinking in round img
use object-fit: cover; in img class ur img will not shrink

**Comment by harshitpaliwal95** [comment]
overall looks good

File: components/Images/images.css (hunk 1 above)
Comment (arulk8) [reply components/Images/images.css:33]: Thanks will do the fix.

**Pull request merged**